content/releases/3-1-0.md

```diff
@@ -0,0 +1,405 @@
+- Change `influxdb.token` to be optional.
+- Change NATS URL configuration to be split between `serverUrl`, `username`, and `password`.
+- Change postgres container to have 1 month default partition size.
+- Change to use the DTO to Domain conversion to create individual objects for the `delete` and `set` operations for `Event` objects.
+- Change value change logging from Trace to Debug.
+
+**Fix**
+- Fix `libreAudit` not accepting <client-name> audience in access token, now accepts both audit and <client-name>.
+- Fix bug preventing messages from being ACK/NAKed when the Audit record was written to Influx.
+- Fix handling for user tag in filter and output.
+- Fix error where `commit_ts` and `uid` in postgres init were `integer` instead of `bigint`.
+
+**Remove** 
+- Removed unused configuration options: `libreDataStoreGraphQL`, `logging.type`, `OpenTelemetry`, and `influxdb`.
+
+### Keycloak Theme
+
+No changes for this release.
+  
+### Router Initilazation
+
+No changes for this release.
+
+## Compatibility
+
+{{< compatible "3.1.0" >}}
+
+## Checksums
+
+{{% checksums "v3.1.0-checksums.txt"  %}}
+
+## Upgrade
+
+{{< reusable/backup >}}
+
+To upgrade to v3.1.0 from v3.0.3, first ensure that you have made corrections to any breaking changes.
+
+### Row Level Access Control
+Rhize now features row level access control. It also requires a different scopemap than previous versions of BAAS. This is a breaking change.
+
+The new scope map features rules and jurisdictions. Jurisdictions can be used to create a group of permissions for certain entities within a hierarhcy scope. This is useful for allowing contract manufacturing organizations to read/write to the data hub, without being able to see the entire manufacturing knowledge graph.
+
+Rhize v3.0.3 will require the following change to your BAAS scopemap in order to be compatible with security going forward:
+- a rule with the `admin` role for specific actions, such as setting up the schema. 
+
+Ensure that your scopemap contains the following rule in some form, with "admin" being case-sensitive:
+
+```json
+{
+    "id": "rule-001",
+    "description": "Admins can do everything",
+    "roles": [
+        "admin"
+    ],
+    "resources": [
+        "*"
+    ],
+    "actions": [
+        "*"
+    ],
+    "jurisdictions": [
+        "*"
+    ]
+}
+```
+
+If you are looking to take advantage of this new functionality see [how to configure your scopemap]({{< relref "row-level-access-control.md" >}}).
+
+### Configuration Harmonization
+Rhize services now use YAML instead of JSON for their configuration files. This is represented in Helm overrides by the object `rhize<SERVICE>Config` (i.e. `rhizeAuditConfig`). Charts for v3.1.0 should only create YAML config files, however there may be differences in the keys for certain configuration options. Consult the values file for each Helm chart and the changelog for specific changes as necessary.
```

Comment:
```suggestion
Rhize services now use YAML instead of JSON for their configuration files. This is represented in Helm overrides by the object `rhize<SERVICE>Config` (i.e. `rhizeAuditConfig`). Charts for v3.1.0 should create only YAML config files, though there may be differences in the keys for certain configuration options. Consult the values file for each Helm chart and the changelog for specific changes as necessary.
```